core/commands/config.go

```diff
@@ -157,7 +163,8 @@ Set the value of the 'Datastore.Path' key:
 //	matchesGlobPrefix("foo.bar.baz", []string{"*", "bar"}) returns true
 //	matchesGlobPrefix("foo.bar", []string{"baz", "*"}) returns false
 func matchesGlobPrefix(key string, glob []string) bool {
-	k := strings.Split(key, ".")
+	normalizedKey, _ := ConfigKeyToLookupData(key)
```

Comment:
strings.EqualFold (below) no longer works, since now it is applied to a rewritten key which has UUIDs.